method/DeepST/DeepST.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+
+# Author_and_contribution: Niklas Mueller-Boetticher; created template
+# Author_and_contribution: ENTER YOUR NAME AND CONTRIBUTION HERE
+
+import argparse
+
+# TODO adjust description
+parser = argparse.ArgumentParser(description="Method ...")
```

Comment:
just write maybe a oneliner

Reply:
I want to give it the same amount of description they gave on their code